src/main.go

```diff
@@ -127,8 +127,38 @@ func main() {
 		fmt.Println(Version)
 		return
 	}
+
+	if *args.Debug {
```

Comment:
is there a reason this is duplicated here rather than to use the flag to return the debug information after everything has been created? There's a lot of duplicate here right now.

Reply:
Not really. At first, It was more different. But since your refactoring, it's the same. So yes, unless we need a specific renderer(which does not exist yet) we can merge them.

Reply:
I don't see a specific renderer coming anytime soon 😅